docs/blog/2023-conference-announcements.rst

```diff
@@ -0,0 +1,109 @@
+.. post:: September 26, 2022
+   :tags: portland-2023, prague, atlantic-2023, newsletter
+
+Announcing Write the Docs conferences for 2023
+==============================================
+
+We're excited to announce two Write the Docs conferences for 2023:
+
+* Write the Docs Portland on May 7-10, 2023, an in person conference in Portland, Oregon.
+* Write the Docs Atlantic, on September 10-12, 2023, a new virtual conference
+  in between European/African and US East coast timezones.
+
+Write the Docs Prague is taking a break this year. Announcements about
+Write the Docs Australia will follow later in 2023.
+
+Write the Docs Portland
+-----------------------
+Write the Docs Portland will be an in person conference, similar to our
+pre-COVID conferences. Speakers will be present in person and speaking
+from the conference stage. We will also offer a free live stream, and remote
+participants are encouraged to participate in our Slack, as for any of
+our previous in person conferences. The conference will follow all local
+COVID regulations that apply at the time. The unconference will not
+be streamed.
```

Comment:
```suggestion
our previous in person conferences. The conference will follow all local
COVID regulations that apply at the time, 
and we will announce more specific policies before our tickets go on sale so people can make an informed decision.
```

Reply:
Adding that sounds good, but did you intentionally remove my sentence about unconference not being streamed?